src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -212,7 +212,7 @@
   <Target Name="CheckTestPlatforms"
           AfterTargets="Test">
     <GetTargetMachineInfo>
-      <Output TaskParameter="TargetOS" PropertyName="TargetOS" />
+      <Output TaskParameter="TargetOS" PropertyName="TargetOS" Condition="'$(TargetOS)' == ''" />
```

Comment:
Change looks good to me although it seems to have failed build.  Let's chat about this when you have a moment as I'm curious why this worked for my purposes during development (i.e. I was able to have clean cross-targeted runs from Windows --> Ubuntu)